indico/modules/events/templates/meta.html

```diff
@@ -2,3 +2,6 @@
 <meta property="og:image" content="{{ event.logo_url if event.has_logo else
                                       (indico_config.IMAGES_BASE_URL + '/logo_indico.png') }}">
 <meta property="og:description" content="{{ event.description|striptags|truncate(500) }}">
+{%- if event.keywords -%}
+<meta keywords="{{ event.keywords|join(',') }}">
```

Comment:
I think the correct syntax is `<meta name="keywords" value="...">`

This line should be indented (we care about readable templates, even if this makes the final HTML output uglier).

Also, I think right now this ends up right behind the `og:description` line without a linebreak, so I would replace the `-%}` with `%}` or `{%-` with `{%}` (the `-` strips whitespace on the side where it is).

Reply:
Done